jenkins_jobs/trigger_weekly_host_os_build.groovy

```diff
@@ -40,11 +40,7 @@ job('trigger_weekly_host_os_build') {
   }
   steps {
     shell(readFileFromWorkspace(
-	    'jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh'))
-    shell(readFileFromWorkspace(
-	    'jenkins_jobs/trigger_weekly_host_os_build/copy_artifacts.sh'))
-    shell(readFileFromWorkspace(
-	    'jenkins_jobs/trigger_weekly_host_os_build/post_build_script.sh'))
```

Comment:
The other two script references should have been removed in the previous commit.